packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -10,21 +10,60 @@
 @import '~carbon-components/scss/globals/scss/layout';
 @import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
 @import '~carbon-components/scss/globals/scss/css--reset';
+@import '~carbon-components/scss/components/ui-shell/ui-shell';
+@import '~carbon-components/scss/components/ui-shell/side-nav';
 
-/// Masthead styles
 /// @access private
-/// @group masthead
+/// @group dotcom ui-shell
+
+$search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
+$search-transition-timing: 95ms;
+
 @mixin masthead {
   .#{$prefix}--header {
+    transition-timing-function: $search-transition;
     background: $ui-02;
-    border-bottom: 1px solid $ui-01;
+    border-bottom: 1px solid $carbon--gray-20;
+
+    .#{$prefix}--header__logo {
+      padding: 0 $spacing-07;
+      margin-right: $spacing-05;
+      position: relative;
+      &::after {
+        content: '';
+        display: block;
+        position: absolute;
+        right: 0;
+        top: 50%;
+        transform: translateY(-50%);
+        height: 1.5rem;
+        width: 0.0625rem;
```

Comment:
I don't know if we want to follow what Carbon does on this, but they use `carbon--rem(1px)` for things like this.

Reply:
Nice thanks